tasks/oracle-db-11-12.yml

```diff
@@ -92,3 +90,45 @@
   tags:
     - skip_ansible_lint
   become: True
+
+- name: "Stop Oracle database"
+  shell: >
+    {{ oracledb.oracle_home }}/bin/dbshut {{ oracledb.oracle_home }}
+  tags:
+    - skip_ansible_lint
```

Comment:
we have an issue in ansible-lint regarding "skip_ansible_lint" https://github.com/lean-delivery/ansible-lint-rules/issues/14 
After implementing CI in this role will fail

Reply:
All skip_ansible_lint tags should be removed

Reply:
all skip_ansible_lint tags have been removed